fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1008,6 +1009,36 @@ public void rebuildIndex(@Nonnull FDBRecordStore store) {
         return buildAsync(this::buildEndpoints, false, additionalLogMessageKeyValues);
     }
 
+    /**
+     * Stop any ongoing online index build (only if it uses {@link SynchronizedSession}s) by forcefully releasing
+     * the lock.
+     * @return a future that will be ready when the lock is released
+     * @see SynchronizedSession#endAnySession(Transaction, Subspace)
+     */
+    public CompletableFuture<Void> stopOngoingOnlineIndexBuildsAsync() {
+        return runner.runAsync(context -> openRecordStore(context).thenApply(recordStore -> {
+            stopOngoingOnlineIndexBuilds(recordStore, index);
+            return null;
```

Comment:
Minor: I believe the inner lambda can be made a `.thenAccept`, and then it doesn't need to return anything.

Reply:
Fixed.